wren-ui/e2e/specs/connectSampleHR.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+import * as helper from '../helper';
+import * as homeHelper from '../commonTests/home';
+import * as modelingHelper from '../commonTests/modeling';
+import { sampleDatasets } from '@/apollo/server/data';
+
+const suggestedQuestions = sampleDatasets.hr.questions;
+
+test.describe('Test HR sample dataset', () => {
+  test.beforeAll(async () => {
+    await helper.resetDatabase();
+  });
+
+  test('Select HR dataset and check suggested questions', async ({ page }) => {
+    await page.goto('/setup/connection');
+    await page.getByRole('button', { name: 'HR' }).click();
```

Comment:
```suggestion
    await page.getByRole('button', { name: 'Human Resource' }).click();
```

I noticed a little weird thing during my local testing. The test will fail, but if changed to 'Human Resource', it appears to succeed. Would you like to help check this on your end?

#### failed
![截圖 2024-10-17 下午2 51 03](https://github.com/user-attachments/assets/5d999f96-6186-494d-a859-e1769ae0f9b9)

#### passed
![截圖 2024-10-17 下午2 52 20](https://github.com/user-attachments/assets/d890c767-1b75-4722-8f6a-ace6eb2b6743)

Reply:
Initially, I too used "Human Resource" as the value for the name(cause that is the text shown in the button), but for some reason, it keeps failing for me locally... Since it works fine for you, maybe we will go ahead with it? 
If possible, maybe someone else could try it locally to double check

Reply:
Hi @saai-syvendra,

Hmm... this seems a bit odd. Let me sync with the team tomorrow first.

Reply:
Hi @saai-syvendra 

I've reviewed the issue mentioned earlier and encountered the same failure at the specified line. So I think we can change it to `Human Resource`. Once that's done, we should be able to merge this PR.
I appreciate your help.

- Failed with `HR`
<img width="998" alt="image" src="https://github.com/user-attachments/assets/d22e0287-7c59-4cda-aad0-d28f90d8273d">

- Success with `Human Resource`
![image](https://github.com/user-attachments/assets/5d3a9b8f-6bba-477f-8b6b-757d2b1f3cae)

Reply:
Hi @andreashimin!

Thank you for the review.. Will make that change